### ISSUE TITLE
`awesome/github` reports incorrectly for URL shorthands

### ISSUE THREAD

**Issue by wooorm** [sindresorhus/awesome-lint#86]
Hello fine folks! 👋

## Problem

I get a message saying `Repository should be on GitHub` ([from here](https://github.com/sindresorhus/awesome-lint/blob/0aafda85989e32f6e2996ffd5947429cb893331b/rules/github.js#L11-L22)) because awesome-lint incorrectly reports my remote URL as not being on GitHub.

The reason for that is that `git config --get remote.origin.url` yields `gh:remarkjs/awesome-remark`.

This is indeed not a valid GitHub URL, but Git supports URL shorthands. I have the following in my Git config (`~/.gitconfig`):

```
[url "git@github.com:"]

	insteadOf = "gh:"
	pushInsteadOf = "github:"
	pushInsteadOf = "git://github.com/"
```

The reason I do this is that it makes it easy to clone from GH: `git clone gh:remarkjs/awesome-remark`.
I don’t know how many people use Git URL shorthands. But this pattern is from [`mathiasbynens/dotfiles`](https://github.com/mathiasbynens/dotfiles) so I’m guessing it’s pretty common.

## Solution

`git remote -v` does resolve URLs, as it gives:

```
origin	git@github.com:remarkjs/awesome-remark (fetch)
origin	git@github.com:remarkjs/awesome-remark (push)
```

[Here’s an example of a remark plugin that parses that result to get a URL](https://github.com/remarkjs/remark-validate-links/blob/fac370a57834fb0cccb7cce4b517eb2b584f718c/lib/find/find-repo.js#L21).

**Comment by transitive-bullshit** [comment]
Thanks @wooorm and @TiagoDanin! 😄
